docker/linux_debian_bullseye_ci

```diff
@@ -11,3 +10,7 @@ RUN \
   /scripts/linux_debian_install_deps.sh && \
   rm /scripts/linux_debian_install_deps.sh && \
   apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ARG DATE IMAGE_NAME
```

Comment:
FWIW I'm not sure it's actually correct to specify multiple ARGs in one line. it's not documented to work.

But that's a separate issue...